### PR TITLE
[#10] Add LimitClause

### PR DESCRIPTION
**Pull request by ejba**

### CODE REVIEW

**Comment by mcserra** [comment]
You'll need to add the limit clause to the `com.dsl.ClauseBuilder` and to the interfaces that allow the limit, in this case, i guess is the return and the with, so add interface contract on AfterWith and create an interface AfterReturn.

File: src/main/java/com/dsl/clauses/LimitClause.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.dsl.clauses;
+
+import com.dsl.AsString;
+
+public class LimitClause implements AsString, Clause {
+  private final int numElements;
+
+  public LimitClause(final int numElements) {
```
Comment (mcserra) [inline src/main/java/com/dsl/clauses/LimitClause.java:11]: The limit clause allows at least a variable as input, so in this case, com.dsl.expressions.param.Variable should be a possible constructor argument. Also, there's the possibility of having an expression on the LIMIT clause like in https://neo4j.com/docs/cypher-manual/current/clauses/limit/#limit-subset-rows-using-expression

**Review by mcserra — CHANGES_REQUESTED** [review]

File: src/main/java/com/dsl/clauses/LimitClause.java (hunk 1 above)
Comment (ejba) [reply src/main/java/com/dsl/clauses/LimitClause.java:11]: Could you help to test the constructor with expression? I don't have a clue how to create such expression.

Comment (mcserra) [reply src/main/java/com/dsl/clauses/LimitClause.java:11]: Will do

**Review by mcserra — APPROVED** [review]

**Pull request merged**